db/scripts/cleanup_scheduled_calls.js

```diff
@@ -13,7 +13,9 @@ async function getDb() {
 async function cleanupScheduledCalls() {
```

Comment:
To avoid any confusion, could we delete the unused `db/cleanup_old_scheduled_calls.sql` ?

Reply:
Done.